### ISSUE TITLE
chrome can't start due to missing fonts

### ISSUE THREAD

**Issue by thedavidmeister** [nix-community/docker-nix#24]
when i try to run karma tests with headless chrome i get

```
08 07 2019 14:20:03.603:ERROR [launcher]: Cannot start ChromeHeadless
        Fontconfig error: Cannot load default config file

DevTools listening on ws://127.0.0.1:9222/devtools/browser/d7d04abe-1c96-416b-87c0-895ef35c235f
Failed to generate minidump.
08 07 2019 14:20:03.603:ERROR [launcher]: ChromeHeadless stdout: 
08 07 2019 14:20:03.603:ERROR [launcher]: ChromeHeadless stderr: Fontconfig error: Cannot load default config file

```

**Comment by zimbatm** [comment]
Here are some hints:

Chrome needs the fontconfig configured on the system. This is basically a registry of all the available fonts. There are more details over here: https://askubuntu.com/questions/492033/fontconfig-error-cannot-load-default-config-file

NixOS provides a way to generate that config file, see nixos/modules/config/fonts/fontconfig.nix. But this is not applicable to the docker image since it's not using NixOS.

Basically you'll have to build your own fontconfig file and then point to it using the FONTCONFIG_PATH environment variable.

**Comment by thedavidmeister** [comment]
yeaaaaaah i swapped to HeadlessFirefox

which has it's own problems on mac, but at least i could CI
